src/components/lib/FlowGrid/gridPoints.js

```diff
@@ -0,0 +1,40 @@
+let upto = (n) => Array.apply(null, {length: n}).map(Number.call, Number)
+
+export default class GridPoint {
```

Comment:
This doesn't warrant a change right now, but I would say that making this a functor as opposed to just having a function in this library that returns another function seems a bit strange if we aren't passing it as props or otherwise checking for equality somewhere.